src/optionals/sqlite/sqlite3.c

```diff
@@ -114499,7 +114499,8 @@ SQLITE_PRIVATE void sqlite3DefaultRowEst(Index *pIdx){
   if( x<99 ){
     pIdx->pTable->nRowLogEst = x = 99;
   }
-  if( pIdx->pPartIdxWhere!=0 ) x -= 10;  assert( 10==sqlite3LogEst(2) );
+  if( pIdx->pPartIdxWhere!=0 ) x -= 10;
```

Comment:
![image](https://user-images.githubusercontent.com/20248039/147679363-3a3a52b7-1282-4302-a00e-db882dd18340.png)

Checking a newly downloaded SQLite amalgamation it seems the assert should actually be within the if statement

```suggestion
  if( pIdx->pPartIdxWhere!=0 ) {x -= 10; assert( 10==sqlite3LogEst(2) );}
```

Thanks for the PR however!

Reply:
Changed :+1:

Reply:
Thank you!